spark-iotdb-connector/src/main/scala/org/apache/iotdb/tsfile/IoTDBOptions.scala

```diff
@@ -0,0 +1,49 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one
+  * or more contributor license agreements.  See the NOTICE file
+  * distributed with this work for additional information
+  * regarding copyright ownership.  The ASF licenses this file
+  * to you under the Apache License, Version 2.0 (the
+  * "License"); you may not use this file except in compliance
+  * with the License.  You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing,
+  * software distributed under the License is distributed on an
+  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  * KIND, either express or implied.  See the License for the
+  * specific language governing permissions and limitations
+  * under the License.
+  */
+package org.apache.iotdb.tsfile
+
+/**
+  * Created by qjl on 16-11-4.
+  */
+class IoTDBOptions(
+                    @transient private val parameters: Map[String, String])
+  extends Serializable {
+
+  val url = parameters.getOrElse("url", sys.error("Option 'url' not specified"))
+
+  val user = parameters.getOrElse("user", "root")
+
+  val password = parameters.getOrElse("password", "root")
+
+  val sql = parameters.getOrElse("sql", sys.error("Option 'sql' not specified"))
+
+  // deprecated:
```

Comment:
why deprecated? since this is the first version, if they are not used, just remove them. Or remove this comment